f/casualtiesCap/f_CasualtiesCapCheck.sqf

```diff
@@ -127,6 +127,9 @@ while {true} do
 		{
 			player sideChat format ["DEBUG (f\casualtiesCap\f_CasualtiesCapCheck.sqf): _started = %1",_started];
 		};
+		if(_started == 0) exitWith {
+			player sideChat format ["DEBUG (f\casualtiesCap\f_CasualtiesCapCheck.sqf): Exiting because no units found"];
```

Comment:
```suggestion
			player sideChat format ["DEBUG (f\casualtiesCap\f_CasualtiesCapCheck.sqf): Exiting because no units found"];
			_end = ""; // Setting _end, such that it does not trigger an ending
```

Reply:
Seems a little hacky. I'd rather there be an explicit flag so changes to the ending bit of the code don't break this.